### PR TITLE
Draggable numbers - fix scale buttons hiding and pressing enter on numbers

### PR DESCRIPTION
**Pull request by dback2**
Addresses:
https://highfidelity.manuscript.com/f/cases/19981/
https://highfidelity.manuscript.com/f/cases/19985/

Pre-Merge Test Plan:
- Enter Interface in desktop mode into any domain with edit permissions
- Open Create and select any entity
- Verify if clicking the Scale number field to edit the number via keyboard that the Rescale and Reset buttons do not hide
- Verify after editing the number in the Scale field via keyboard and pressing enter that it returns to it's previous state with the step arrows visible and number not highlighted

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15309/HighFidelity-Beta-PR14448-8610247.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15309/HighFidelity-Beta-PR14448-8610247.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15309/HighFidelity-Beta-Interface-PR14448-8610247.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15309/HighFidelity-Beta-PR14448-8610247.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15309/HighFidelity-Beta-Interface-PR14448-8610247.exe

**Comment by QliCknox** [comment]
Currently Testing

File: scripts/system/html/js/draggableNumber.js [hunk 1]
```diff
@@ -130,6 +130,12 @@ DraggableNumber.prototype = {
         this.elText.style.visibility = "visible";
     },
     
+    keyPress: function(event) {
+        if (event.keyCode === 13) {
```
Comment (huffman) [inline scripts/system/html/js/draggableNumber.js]: Can we put 13 in a `const`, to make it more clear that it is the enter key?

Comment (huffman) [reply scripts/system/html/js/draggableNumber.js]: Oops, Thoys got to it before me.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15529/HighFidelity-Beta-PR14448-8610247.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15529/HighFidelity-Beta-PR14448-8610247.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15529/HighFidelity-Beta-Interface-PR14448-8610247.exe

**Comment by QliCknox** [comment]
Pass

PR retains passes from prior prs

**Pull request merged**